bindgen/Context/Function.zig

```diff
@@ -430,6 +437,62 @@ pub const Parameter = struct {
     }
 };
 
+const return_type_init_map: StaticStringMap([]const u8) = .initComptime(.{
+    // builtins
+    .{ "Vector2", ".zero" },
+    .{ "Vector3", ".zero" },
+    .{ "Vector4", ".zero" },
+    .{ "Vector2i", ".zero" },
+    .{ "Vector3i", ".zero" },
+    .{ "Vector4i", ".zero" },
+    .{ "Basis", ".identity" },
+    .{ "Transform2D", ".identity" },
+    .{ "Transform3D", ".identity" },
+    .{ "Projection", ".identity" },
```

Comment:
do we need to  add `Variant`/`.nil`?

Reply:
Yep good one. Updated https://github.com/gdzig/gdzig/pull/81/commits/ddbc320dd12bd52ee5d252866ac580d3167df125